security/backend/file_management/README.md

```diff
@@ -0,0 +1,28 @@
+# File Management
+
+**System configuration files must have restricted access.**
+
+This means
+
+1. They are not accessible in any way from the internet
+2. They are not accessible by insiders who have no need to access them
```

Comment:
What is an insider?

Reply:
An insider would be someone who's part of the organization or team